examples/provider-verification/main.tf

```diff
@@ -7,7 +7,13 @@ terraform {
 }
 
 provider "altitude" {
-  client_id     = "{{CLIENT ID}}"
-  client_secret = "{{CLIENT SECRET}}"
-  mode          = "Local"
+  client_id     = "sJ8kVG1yCFyD8qOTLAS5yEy2F28ZM7qO"
+  client_secret = "w455Q1EFsVkOKxm4u05Reuiy84mp1Ka3ihHSfHBctHvxNtzZ61W-HWxxETHzsfZN"
```

Comment:
Is there a way to pass the id and secret down without hardcoding it?

Reply:
yeah thats not meant to be there - will delete this client